package.json

```diff
@@ -1,39 +1,40 @@
 {
-    "name": "wetzel",
```

Comment:
Due to all the spacing changes, it's hard to tell from this diff that the only change was the addition of this devDependency: `"json-schema-test-suite": "0.0.10"`.  Any chance you can fix the file changes so that it's more clear?  That being said, I also don't see how it's being used at all.  Should this change even be part of this commit?